### PR TITLE
Fake dates with GZDates 

### PR DESCRIPTION
**Pull request by mariabernis**
#### :tophat: What? Why?

Added new GZDates class to create random dates. Can easily create dates around current day, or fordward/backward or around a specific date. 
#### :clipboard: Developer checklist
- [x] Unit Tests (if necessary)
- [x] Documentation

### CODE REVIEW

File: Gizou/GZDates.m [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
++ (NSDate *)birthday
+{
+    return [self birthdayForAgesBetween:18 and:65];
+}
+
++ (NSDate *)birthdayForAgesBetween:(NSUInteger)minAge and:(NSUInteger)maxAge
+{
+    NSDate *today = [NSDate date];
+    NSCalendar *calendar = [self _calendar];
+    NSDateComponents *fromComponents = [[NSDateComponents alloc] init];
```
Comment (sergigracia) [inline Gizou/GZDates.m]: :new: ?

**Comment by sergigracia** [comment]
:+1:

File: Gizou/GZDates.m [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    }
+    
+    return [self days:daysRange around:referenceDate];
+}
+
++ (NSDate *)days:(NSInteger)days from:(NSDate *)referenceDate
+{
+    NSInteger s = signbit(days) == 0 ? 1 : -1;
+    NSCalendar *calendar = [self _calendar];
+    NSDateComponents *fromComponents = [[NSDateComponents alloc] init];
```
Comment (isaacroldan) [inline Gizou/GZDates.m]: `new` everywhere

**Comment by isaacroldan** [comment]
:+1:

**Pull request merged**